UnityProject/Assets/Scripts/GameModes/NukeOps.cs

```diff
@@ -29,14 +29,20 @@ public override void StartRound()
 		base.StartRound();
 	}
 
+	public override bool IsPossible()
+	{
+		return FindObjectOfType<Nuke>() != null ? true : false;

```

Comment:
Just one last thing, make sure to include the base checks as well:
```suggestion
		return base.IsPossible() && (FindObjectOfType<Nuke>() != null);
```